yb_migrate/cmd/exportData.go

```diff
@@ -204,3 +200,11 @@ func checkDataDirs() {
 		}
 	}
 }
+
+func createExportDataDoneFlag() {
+	exportDoneFlagPath := exportDir + "/metainfo/flags/exportDataDone"
+	err := exec.Command("touch", exportDoneFlagPath).Run() //to inform import data command
+	if err != nil {
+		utils.ErrExit("couldn't touch file %q: %v", exportDoneFlagPath, err)
+	}
```

Comment:
`os.Create()` (https://pkg.go.dev/os#Create) would have been simpler approach to create a new empty file.

Reply:
done